__tests__/__snapshots__/sources.ts.snap

```diff
@@ -128,19 +128,6 @@ Object {
 }
 `;
 
-exports[`query source current member should return current member as blocked 1`] = `
```

Comment:
Replaced with a better expect statement.

Reply:
NICE! I love this :D